source/common/config/utility.h

```diff
@@ -212,6 +212,15 @@ class Utility {
    * @return std::string resource name.
    */
   static std::string resourceName(const ProtobufWkt::Any& resource);
+
+  /**
+   * Creates the set of stats tag extractors requested by the config and transfers ownership to the
+   * caller.
+   * @param bootstrap bootstrap proto
+   * @return Owning vector of TagExtractors
```

Comment:
Oh, I wasn't aware of this.  Thanks for the heads up.  Two questions:
1.  This is pretty inconsistent in Envoy now.  Should we clean this up?
2.  We don't actually render the doxygen, do we?  If we do, IIRC, the return type is provided there automatically.  If we don't, we're just typing out the type that could be read a few lines below.  Is there an advantage to that?  And why don't we do this for @params?  I'm not necessarily suggesting we do it a different way, just curious about the reasoning as it's not immediately obvious to me.

Reply:
TBH I think this was probably a habit picked up on some project I worked on long ago which I no longer remember. It may very well make no sense. I'm not sure how easy it would be to clean up. In a perfect world we would actually run doxygen if even for the only reason to check that comments are compliant and fail if they are not. (there is an issue on this)

Reply:
Gotcha.  I won't worry about it for now.  I was just curious.